microcosm_flask/templates/swagger_ui.py

```diff
@@ -0,0 +1,42 @@
+# flake8: noqa
+html = """
+<!-- HTML for static distribution bundle build -->
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Swagger UI</title>
+    <!-- <link rel="stylesheet" type="text/css" href="./swagger-ui.css" /> -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/swagger-ui-dist@4.12.0/swagger-ui.min.css">
```

Comment:
Still need to swap these to the `static.globality.com` resources and add the `integrity` checks.

Reply:
Done